widget/entry_test.go

```diff
@@ -12,6 +12,20 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestEntry_InterfaceCompatibilityTest(t *testing.T) {
+	entry := fyne.Widget(NewEntry())
+	var ok bool
+
+	_, ok = entry.(fyne.Draggable)
```

Comment:
The model used elsewhere is to put this alongside the type definition.
Perhaps it would be worth following that approach so it is caught at compile time - for example see canvas/canvas.go